Gemfile

```diff
@@ -25,6 +25,13 @@ group :test do
   gem 'rspec', '~> 3.9'
 end
 
+gem "rack", ENV['RACK_VERSION']
+if ENV['RACK_VERSION'] == '2.0'
```

Comment:
Versions can be compared using: https://stackoverflow.com/a/3064161

Reply:
Good point, updated - thanks!